resources/views/page-builder/image_text.blade.php

```diff
@@ -1,4 +1,4 @@
-<div class="mx-auto mb-12 flex max-w-7xl flex-col gap-20 px-6 md:mb-32 md:gap-28"{{ ($component_id ?? false) ? " id=" . $component_id : "" }}>
+<div class="mx-auto flex max-w-7xl flex-col gap-20 px-6 component md:gap-28"{{ ($component_id ?? false) ? " id=" . $component_id : "" }}>
```

Comment:
Maybe we can just do `id="$component_id ?? uniqueid()"` There might be a random `0` in html attributes now.